storage/innobase/xtrabackup/doc/source/using_xtrabackup/privileges.rst

```diff
@@ -131,3 +130,10 @@ to full backups would be:
    mysql> GRANT BACKUP_ADMIN, PROCESS, RELOAD, LOCK TABLES, REPLICATION CLIENT ON *.* TO 'bkpuser'@'localhost';
    mysql> GRANT SELECT ON performance_schema.log_status TO 'bkpuser'@'localhost';
    mysql> FLUSH PRIVILEGES;
+
+``SELECT`` privilege on the `keyring_component_status table <https://dev.mysql.com/doc/refman/8.0/en/performance-schema-keyring-component-status-table.html>`__  to view the attributes and status of the installed keyring component when in use.
```

Comment:
seems like we are missing a `*` in the begining

Reply:
fixed